app/models/qdm/patient_characteristic_sex.rb

```diff
@@ -0,0 +1,4 @@
+class QDM::PatientCharacteristicSex
+  include Mongoid::Document
+  field :qdm_version, type: String, default: '5.3'
```

Comment:
Should this include "  field :category, type: String, default: 'patient_characteristic'"?

Reply:
Per J.B.'s point above, yes, all of these will be `patient_caracteristic`s.